app/com/lucidchart/open/cashy/amazons3/S3Sync.scala

```diff
@@ -56,7 +56,7 @@ class S3Sync extends CloudfrontConfig {
   private def checkKrakenQuota() {
     val usageThreshold = configuration.getDouble("kraken.usageAlertThreshold").get
     val usage = KrakenClient.checkQuota()
-    if (!usage.isDefined || usage.get > usageThreshold) {
+    if (usage.isDefined && usage.get > usageThreshold) {
```

Comment:
The Paul, thank you for teaching me the error of my ways. I'm sorry to have disgusted you. I will always use `.exists` from now on!